### ISSUE TITLE
[Beetle PSX HW] bad render in vulkan: Jumping Flash 2 NTSC.

### ISSUE THREAD

**Issue by darkxex** [libretro/beetle-psx-libretro#481]
Hello, Vulkan have a bug in render this game, please check this.
Game: Jumping Flash! 2. [NTSC]

Vulkan render:
![vulkan](https://user-images.githubusercontent.com/17991404/51221191-3e334380-1917-11e9-9992-2181877ef7b2.png)
Software render:
![software](https://user-images.githubusercontent.com/17991404/51221273-9ec28080-1917-11e9-999b-8b17b232ad7a.png)

**Comment by twinaphex** [comment]
Yeah, I can confirm this is an issue.

@Themaister Got any ideas about this?

**Comment by hizzlekizzle** [comment]
Does the line-to-quad hack affect this at all? That is, if you change it to 'disabled' does the issue persist?

**Comment by darkxex** [comment]
the problem always persists.

**Comment by darkxex** [comment]
3/13/2020 the problem still persist.

**Comment by rz5** [comment]
Seems to be related to the dithering pattern option. When the dither pattern is set to 1x or made to scale with internal resolution, I get the bug.

If the option is set to off, it seems to render fine.

**Comment by darkxex** [comment]
with dithering off, still i have the bug.

**Comment by darkxex** [comment]
the only way that the bug does not appear is in software render.

**Comment by rz5** [comment]
Right, in my past post I was using the GL renderer. In it, disabling the dither mask fixes the bug. So this happens on both Vulkan and GL.

With the vk renderer, changing the texture filtering will alter the colors of the graphics bug but there seems to be no way to work around the bug with the current core options.
